index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```

Comment:
this is a suggesstion.
```suggestion
  </tr>
```
this is me editing the suggestion